server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/api.ts

```diff
@@ -129,6 +135,20 @@ export function create(
 		},
 	);
 
+	router.post("/broadcast-signal", (req, res) => {
+		try {
+			const tenantId = req.body?.tenantId;
+			const documentId = req.body?.documentId;
+			const signalContent = req.body?.singalContent;
+			const signalRoom: IRoom = { tenantId, documentId };
+			const payload: IBroadcastSignalEventPayload = { signalRoom, signalContent };
+			collaborationSessionEventEmitter.emit("broadcast-signal", payload);
+			res.status(200).send("Triggering debug signal from tinylicious");
```

Comment:
This message says `tinylicious` but this is `routerlicious`. I think, though, it would be OK to just send the message "OK", so that this isn't worded specifically to Routerlicious. For example, would look weird if someone contacting AFR received something about "routerlicious"